Public/Src/Engine/Processes/SandboxedProcess.cs

```diff
@@ -81,11 +81,11 @@ internal SandboxedProcess(SandboxedProcessInfo info)
                 s_binaryPaths = new BinaryPaths(); // this can take a while; performs I/O
             }
 
-            // If unspecified make the injection timeout 10 mins. Also, make it no less than 10 mins.
-            m_timeoutMins = info.Timeout.HasValue ? ((uint)info.Timeout.Value.TotalMinutes) : 10;
-            if (m_timeoutMins < 10)
+            // If unspecified make the injection timeout 15 mins. Also, make it no less than 15 mins.
+            m_timeoutMins = info.Timeout.HasValue ? ((uint)info.Timeout.Value.TotalMinutes) : 15;
+            if (m_timeoutMins < 15)

```

Comment:
can you use `DefaultProcessTimeout` from `UnSandboxedProcess.cs`? (or the other way around)